charts/gsp-cluster/templates/04-main-team-pipelines/cd-smoke-test-pipeline.yaml

```diff
@@ -95,13 +110,17 @@ spec:
               - -c
               - |
                 echo "generating helm values for latest image versions..."
+                IMAGE_TAG_TYPE=$(cat ecr/digest | cut -d ':' -f 1)
+                IMAGE_TAG=$(cat ecr/digest | cut -d ':' -f 2)
+                IMAGE_REPOSITORY="${IMAGE_URI}@${IMAGE_TAG_TYPE}"
```

Comment:
🤔 

I hadn't seen this hack before, but it's *horrific*

Reply:
it's from the previous code

Reply:
Yeah from existing code... I factored  it out like that so that script would error out (which it previously did not)